.eslintrc

```diff
@@ -30,7 +30,7 @@
     "no-shadow": 2,
     "no-shadow-restricted-names": 2,
     "no-spaced-func": 2,
-    "no-trailing-spaces": 2,
+    "no-trailing-spaces": 0,
```

Comment:
For some reason mine ignores this `.editorconfig` rule completely 😢. Reverted: fabc6d8eaba7904ae667dd20741bed9ea1a39ca0